app/src/main/java/de/westnordost/streetcomplete/quests/parcel_locker_mail_in/AddParcelLockerMailIn.kt

```diff
@@ -0,0 +1,33 @@
+package de.westnordost.streetcomplete.quests.parcel_locker_mail_in
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.quests.YesNoQuestForm
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddParcelLockerMailIn : OsmFilterQuestType<Boolean>() {
+
+    override val elementFilter = "nodes with amenity = parcel_locker and !parcel_mail_in"
+    override val changesetComment = "Specify if it's possible to drop off parcels with this locker"
+    override val wikiLink = "Tag:amenity=parcel_locker"
+    override val icon = R.drawable.ic_quest_notes
+    override val isDeleteElementEnabled = true
+    override val achievements = listOf(CITIZEN)
```

Comment:
Maybe this should rather count towards the `POSTMAN` achievement?

Reply:
Done ^^.